src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionMethodLengthTest.java

```diff
@@ -0,0 +1,134 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package org.checkstyle.suppressionxpathfilter;
+
+import static com.puppycrawl.tools.checkstyle.checks.sizes.MethodLengthCheck.MSG_KEY;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.checks.sizes.MethodLengthCheck;
+
+public class XpathRegressionMethodLengthTest extends AbstractXpathTestSupport {
+
+    private final String checkName = MethodLengthCheck.class.getSimpleName();
+
+    @Override
+    protected String getCheckName() {
+        return checkName;
+    }
+
+    @Test
+    public void testOne() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionMethodLengthOne.java"));
+
+        final DefaultConfiguration moduleConfig = createModuleConfig(MethodLengthCheck.class);
+        moduleConfig.addProperty("max", "10");
+
+        final String[] expectedViolations = {
+            "4:5: " + getCheckMessage(MethodLengthCheck.class, MSG_KEY,
+                    11, 10, "SuppressionXpathRegressionMethodLengthOne"),
+        };
+
+        final List<String> expectedXpathQueries = Arrays.asList(
+                "/COMPILATION_UNIT/CLASS_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]"
+                        + "/OBJBLOCK/CTOR_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]",
+                "/COMPILATION_UNIT/CLASS_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]"
+                        + "/OBJBLOCK/CTOR_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]/MODIFIERS",
+                "/COMPILATION_UNIT/CLASS_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]"
+                        + "/OBJBLOCK/CTOR_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionMethodLengthOne']]"
+                        + "/MODIFIERS/LITERAL_PROTECTED"
+        );
+
+        runVerifications(moduleConfig, fileToProcess, expectedViolations, expectedXpathQueries);
+    }
+
+    @Test
+    public void testTwo() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionMethodLengthTwo.java"));
+
+        final DefaultConfiguration moduleConfig = createModuleConfig(MethodLengthCheck.class);
+        moduleConfig.addProperty("max", "5");
+        moduleConfig.addProperty("countEmpty", "false");
+        moduleConfig.addProperty("tokens", "METHOD_DEF");
+
+        final String[] expectedViolations = {
+            "15:5: " + getCheckMessage(MethodLengthCheck.class, MSG_KEY,
+                    6, 5, "methodOne"),
```

Comment:
Please keep it one same line, if line length check permits. Same for other `expectedViolations`

Reply:
done. One method is exceeds the line length.